src/tribler/core/components/libtorrent/torrentdef.py

```diff
@@ -237,19 +240,19 @@ def set_tracker(self, url):
             url = url[:-1]
         self.torrent_parameters[b'announce'] = url
 
-    def get_tracker(self):
+    def get_tracker(self) -> Optional[str]:
         """
         Returns the torrent announce URL.
         """
         return self.torrent_parameters.get(b'announce', None)
 
-    def get_tracker_hierarchy(self):
+    def get_tracker_hierarchy(self) -> List:
```

Comment:
It is possible to define the resulted type more strictly as `List[List[str]]`
```suggestion
    def get_tracker_hierarchy(self) -> List[List[str]]:
```
It is a list of "tiers", where each tier is a list of torrent URLs

See: https://github.com/rakshasa/libtorrent/blob/master/doc/multitracker-spec.txt